src/examples/zkapps/hello_world/run_live.ts

```diff
@@ -0,0 +1,93 @@
+// Live integration test against real Mina network.
+import { AccountUpdate, Field, Mina, PrivateKey, fetchAccount } from 'snarkyjs';
+import { HelloWorld, adminPrivateKey } from './hello_world.js';
+
+const useLocalNetwork = process.env.USE_LOCAL_NETWORK === 'true';
+const senderKey = useLocalNetwork
```

Comment:
Can you share some light on the stability of using the local network for all use cases? If we want devs to start migrating to this asap, we could highlight this in docs somewhere (doesn't have to be this PR). Otherwise, we can leave it out for now and come up with better DX?

Reply:
Oh and one more thing. In order to support the complete dev/test flow, we also need to add one more feature to Docker Image - the RDBMS+ArchiveNode so that zkApp devs using SnarkyJS can also work with events/actions.